src/simtools/runners/corsika_simtel_runner.py

```diff
@@ -157,8 +157,10 @@ def _make_run_command(self, run_number=None, input_file=None):
         except TypeError:  # allow for self.label to be None
             weak_pointing = False
 
+        self.corsika_config.array_model.export_all_simtel_config_files()
```

Comment:
I am worried that this called from `corsika_cofnig` rather than from the `simulator_array`, but it might be OK. We will see once I merge this to the multipipe branch.

Reply:
Ok, we wait and see if it is an issue?

Reply:
Yes, we'll see later.